webapp/src/hooks/hooks.js

```diff
@@ -44,7 +44,7 @@ export default class Hooks {
             return Promise.resolve({});
         }
 
-        if (message && (message.startsWith('/jira subscribe ') || message === '/jira subscribe')) {
+        if (message && message.startsWith('/jira subscribe') && !message.startsWith('/jira subscribe list')) {
```

Comment:
This is unfortunate, that we'd need to list any other special params here. Isn't the client-side hook just for `^/jira subscribe$`?

Reply:
Yes, I was cleaning up the redundancy here. Some user input like `"/jira subscribe "` would then not work if we change to be an exact match.

@cpoile Do you have thoughts on how we can elegantly handle the case where we don't want to open the subscriptions modal if the user types `/jira subscribe list`?

I just realized I've introduced a bug here too. `/jira subscribesometext` would also trigger the subscriptions modal.

Reply:
trim white space, then exact match?

Reply:
Sounds good